index.bs

```diff
@@ -320,18 +315,18 @@ The <dfn method for=IdleDetector>start(|options|)</dfn> method steps are:
     <wpt>
       interceptor.https.html
     </wpt>
-1.  [=Queue a global task=] on the [=relevant global object=] of [=this=] using
-    the [=idle detection task source=] to perform the following steps, but
-    [=abort when=] |this|.{{IdleDetector/[[state]]}} becomes `"stopped"`.
-    1.  Let |permissionState| be the [=permission state=] the [=powerful
-        feature=] named `"idle-detection"`.
+1. [=In parallel=]:
+    1.  Let |permissionState| be the [=permission state=] of `"idle-detection"`.
```

Comment:
Do we need to be "in parallel" to check the permission state? The way the Permissions specification is written it doesn't imply an asynchronous operation but in Chromium it does have to happen asynchronously because the browser process is the source of truth for permission state. Given that, there's nothing in these steps which would block the main thread from a specification perspective. It is technically possible for scripts to observe the difference between method steps which occur before or after the current microtask checkpoint by observing `Promise` resolution behavior. Is it important for specifications to be perfectly clear on that or can it be an implementation detail?

Reply:
> Do we need to be "in parallel" to check the permission state?

You are absolutely correct. I added it as a "foot-gun prevention": if someone comes along later to make modification and/or doesn't read the Permission spec, they may wrongly assume it's all sync. Generally speaking, I think the permission state checks in every browser is always in parallel for the reason you mentioned (the truth is in an internal browser database or file or possibly gotten from the OS). 

 > Given that, there's nothing in these steps which would block the main thread from a specification perspective.

Acknowledged. It's your call given my rationale above. I can remove "[=in parallel=]:", no problem.   

> Is it important for specifications to be perfectly clear on that or can it be an implementation detail?

I think it's important just as an editorial aid (and an aid to the reader). But it's definitely an implementation detail how the permission state is retrieved and the subsequently resolved (task queuing aside, etc.).

Reply:
I think the defense in depth here is worthwhile and acknowledges the reality of how most engines implement permissions. This should probably be mentioned in the Permissions spec.